src/bdfsclient/ActionHandler.cpp

```diff
@@ -148,7 +148,8 @@ namespace dfs
     }
 
     // Cache size set to 100MB / (blockSize * (dataCount + codeCount)), flushing every 10 seconds
-    volume->EnableCache(std::make_unique<dfs::Cache>("cache", volume.get(), 200, 10));
+    std::string cacheDir = "/etc/drive/" + name + "/" + "cache";
```

Comment:
As a practice, we should store the program-generated data in /var/drive instead of /etc. /etc should be used for configurations.